articles/concepts/dirac-notation.md

```diff
@@ -25,42 +25,46 @@ More generally, if $\psi$ and $\phi$ are quantum state vectors their inner produ
 The following convention is used to describe the quantum states that encode the values of zero and one (the single-qubit computational basis states):
 
 $$
-\begin{bmatrix} 1 \\\\  0 \end{bmatrix} = |0\rangle,\qquad
-\begin{bmatrix} 0 \\\\  1 \end{bmatrix} = |1\rangle.
+\begin{bmatrix} 1 \\\\  0 \end{bmatrix} = \ket{0},\qquad
+\begin{bmatrix} 0 \\\\  1 \end{bmatrix} = \ket{1}.
 $$
 
-The following notation is often used to describe the states that result from applying the Hadamard gate to $|0\rangle$ and $|1\rangle$ (which correspond to the unit vectors in the $+x$ and $-x$ directions on the Bloch sphere):
+### Example: representing the Hadamard operation with Dirac notation
+
+The following notation is often used to describe the states that result from applying the Hadamard gate to $\ket{0}$ and $\ket{1}$ (which correspond to the unit vectors in the $+x$ and $-x$ directions on the Bloch sphere):
 
 $$
-\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}=H|0\rangle = |{+}\rangle,\qquad
-\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  -1 \end{bmatrix} =H|1\rangle = |{-}\rangle .
+\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}=H\ket{0} = |{+}\rangle,\qquad
```

Comment:
There are still a lot of `|...\rangle` left in the formulas

Reply:
I'm not surprised, there's a lot of different variations that make it hard to find and replace. Thanks for catching this one!